### ISSUE TITLE
[Assistive tech and device support] Focus state isn't visible. (11.09.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#33342]
#### VFS team name
Decision Tools

#### VFS product name
Find a VA Form PDF Download Instructions

#### Point of Contact/Reviewers
Josh Kim (@joshkimux) - Accessibility

---

### Platform Issue
Focus state isn't visible.

### Issue Details
Focus is lost when the user closes the modal. Keyboard users will need to tab all the way back to where they were from the top of the page and screen reader users will be left confused.

### VA.gov Experience Standard
[Category Number 11, Issue Number 09](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.7_AA

---

### Platform Recommendation
Focus should return to the element that triggered the modal when it is closed. In this case, that would be the download link.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by srsuddath** [comment]
@joshkimux this fix is up on staging. you should be able to close the modal and have focus return the the link that you clicked to open it

**Comment by joshkimux** [comment]
https://user-images.githubusercontent.com/14154792/147113226-244a497a-dd81-424d-967c-24b24bf71feb.mov

Perfect @srsuddath 🥳 

One minor thing I could use verification on-- it looks like the modal doesn't open when I:
1. open an incognito window in Chrome
2. paste in `https://staging.va.gov/find-forms/?q=10-10ez` to the url
3. tab to the download link
4. press `enter`

Would you be able to replicate that issue on your end?

**Comment by srsuddath** [comment]
I will look into it! it might be that the click handler isnt registering the keyboard event. I will look into it!

**Comment by srsuddath** [comment]
looks like I need to add an event handler so that we can use enter press on those links as well, will have a fix shortly

**Comment by srsuddath** [comment]
@joshkimux the fix for this PR is now live on staging, if you wouldn't mind checking it out! enter key should work to open the modal as well!

**Comment by joshkimux** [comment]
Just verified! Works perfectly 💯
